### DISCUSSION TITLE
Export to CSV  feature in the Traces

### DISCUSSION THREAD

**Discussion by Aniket3007** [langfuse/langfuse#2456] [Ideas]
### Describe the feature or potential improvement

More refined as one can easily get all the traces into a csv file along with the scores  instead of taking "generations" into account.
So the scores for each trace can also be a field on the CSV file.
If i want to add the same scores to generation page, which has "Export to CSV" feature
we need to extract  the corresponding  generation_id  and add it as a field in langfuse.score() method.
Also to get generation_id I couldnt find any method to access  corresponding generation_id like we have for trace_id which is trace_id = langfuse_handler.get_trace_id()  .

**Comment by marcklingen** [comment]
Hi @Aniket3007, this is on our backlog, thanks for adding your plus one to this.

In the meantime, you can pull traces via the Python sdk, see more details here: https://langfuse.com/docs/query-traces
